const throwError = () => {
    process.stdout.write(`Error`)
    process.exit(1)
}

module.exports = throwError